### PR TITLE
harmonise chart metadata / define proper versions

### PR DESCRIPTION
**Pull request by xrstf**
**What this PR does / why we need it**:
This is primarily a spring cleanup:

* `apiVersion` is a mandatory field in Chart.yaml files.
* All charts have a (somewhat) more meaningful version number.
* Maintainer, homepage and sources updated.

---

I propose that we give our charts a meaningful version that we actually update with each revision (on a best-effort basis; kinda not sure how often we would have parallel PRs for the same charts). As Helm versions must be SemVer2 compatible, this means

* no leading "v" prefix
* "a.b.c-d" means "is a pre-release", not "is a patch-release", i.e. `1.0.0 > 1.0.0-1`.

I think this will make a `helm ls` much more meaningful, especially for clusters not permanently kept up-to-date like ours.

**Release note**:
```release-note
improve Helm charts metadata to make Helm-based workflows easier and aid in cluster updates
```

### CODE REVIEW

File: api/hack/ci/ci-sync-charts.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@ apk add --no-cache -U git bash openssh
 cd $(dirname $0)/../../..
 git fetch
 export LATEST_VERSION=$(git describe --tags --abbrev=0)
-sed -i "s/__KUBERMATIC_TAG__/$LATEST_VERSION/g" config/kubermatic/values.yaml
+sed -i "s/__KUBERMATIC_TAG__/$LATEST_VERSION/g" config/kubermatic/values.yaml config/kubermatic/Chart.yaml
```
Comment (xrstf) [inline api/hack/ci/ci-sync-charts.sh:9]: I have no idea if this works. `sed --help` mentioned multiple input files are okay.

File: config/minio/Chart.yaml [hunk 2]
```diff
@@ -1,13 +1,16 @@
+apiVersion: v1
 name: minio
-version: 1.0.0
+version: 2019.4.9-1.22.30-1
```
Comment (xrstf) [inline config/minio/Chart.yaml]: I know, this is ugly/cheating, but ... it works I guess.

File: api/hack/ci/ci-sync-charts.sh (hunk 1 above)
Comment (xrstf) [reply api/hack/ci/ci-sync-charts.sh:9]: Well, works on my machine.

File: config/minio/Chart.yaml (hunk 2 above)
Comment (kron4eg) [reply config/minio/Chart.yaml]: would it make sent to adopt upstream style of version naming, like latest release is `RELEASE.2019-04-09T01-22-30Z`? Will it even work or helm would just spit an error?

File: config/logging/fluentbit/Chart.yaml [hunk 3]
```diff
@@ -1,13 +1,15 @@
...
+version: 1.0.4-1
+appVersion: 1.0.4
 description: fluentbit
 keywords:
-- k8s
 - kubermatic
+- logging
 - fluentbit
-home: http://www.kubermatic.io/
+home: http://fluentbit.org/
```
Comment (kron4eg) [inline config/logging/fluentbit/Chart.yaml]: this site announces latest release as v0.12.17 made a year ago... Looks like it's abandoned and https://fluentbit.io/ is used instead

File: config/minio/Chart.yaml (hunk 2 above)
Comment (xrstf) [reply config/minio/Chart.yaml]: Well the Helm docs clearly state that versions must be SemVer 2 and SemVer 2 is pretty strict about its `X.Y.Z-a.b.c+d.e.f` system.

**Review by kron4eg — APPROVED** [review]
/lgtm
/approve

**Pull request merged**